stories/Molecules/Blocks/Lightbox/lightbox.scss

```diff
@@ -0,0 +1,43 @@
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/mixins";
+
+/* Modal lightbox card start */
+.modal {
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
+  display: flex;
+  position: fixed;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  background: rgba(247, 247, 247, 0.8);
+}
+
+.modal-content {
+  width: 100%;
+  max-width: 837px;
+  height: 492px;
```

Comment:
@harishkumar-srijan It should be in vh or in % otherwise it won't be consistent in all the devices

Reply:
Done